### ISSUE TITLE
Update requirements.md file

### ISSUE THREAD

**Issue by KirstieJane** [KirstieJane/BrainsForPublication#19]
There is a [requirements.md]() file inside the [docs]() folder. It should contain all the packages and downloads that anyone needs to run the code in the scripts directory.

**Comment by amitkumarj441** [comment]
Hey @KirstieJane ,

I gone through your scripts directory and found that all the necessary packages and downloads are in the [DESCRIPTION_DATA](https://github.com/KirstieJane/DESCRIBING_DATA/wiki/Making-Surface-Plots-with-Pysurfer) repo! But this wiki should be in the wiki page of BrainsForPublication repo which is not there! What type of documentation you're looking for? I'm totally confused!!

Thank you
Amit Kumar Jaiswal

**Comment by KirstieJane** [comment]
Hi @amitkumarj441 - yep - the documentation needs to be moved over from that repo to this one and then we need to make more documentation for the code we wrote recently.

Ideally we'll end up with documentation that says what the requirements are ([docs/requirements.md](https://github.com/KirstieJane/BrainsForPublication/blob/master/docs/requirements.md)) and then separate documentation for each of the commands (eg: [docs/mni_glass_brain.md](https://github.com/KirstieJane/BrainsForPublication/blob/master/docs/mni_glass_brain.md)).

**Comment by amitkumarj441** [comment]
@KirstieJane Got your point! It's time to initiate a PR into docs for each commands! Right?
Need to know, where to start from!

**Comment by KirstieJane** [comment]
Hi @amitkumarj441 - I'm not sure that this is the most useful issue for you to help with - most of the documentation hasn't been written yet! And unfortunately, the usage is specialised for neuroimagers so I don't think you'd be able to write it.

Thank you for the enthusiasm though.

**Comment by amitkumarj441** [comment]
Yes @KirstieJane ! You're right! I'm from a different background(Computer Science), but I'm able to help you in code, I know python and use to write code in C++,Python,Scala,Java,R,Matlab etc. The only thing I need here is the whole question with explanation and then I'll able to code for this! 
Well, I'm still quite a learner and this semester I have taken Machine Learning, AI, Data Mining etc. courses and hence I want to delve myself deeper into this field.

Thank you
Amit Kumar Jaiswal

**Comment by KirstieJane** [comment]
That's really great @amitkumarj441 but the goal of this project is to build very simple code to help neuroscientists who are too busy or not confident in their coding skills to make figures for their scientific publications. While anyone with any level of coding experience can contribute to the project, there's quite a lot of expert knowledge around brain imaging methods that are necessary to understand the motivations for the tools we're providing. I'm not able to provide that background knowledge as a mentor for this project - it would take too much time (on the order of 3 years of a PhD!)

**Comment by amitkumarj441** [comment]
Alright @KirstieJane !!
 
I know theoretical knowledge of Magnetic Resonance Imaging(MRI), but i never implemented this! Can i start writing a script for this in scala(this needs sbt) and python!

**Comment by KirstieJane** [comment]
Hi @amitkumarj441 - you absolutely can write any code to play with MRI data, but I won't merge it into this project. The goal is to keep it very simple and well constrained.

Do feel free to fork the repo and build from what we've done if you'd like to!

**Comment by amitkumarj441** [comment]
@KirstieJane If you don't mind, I just found your repository [NSPN_MRIProcessing](https://github.com/KirstieJane/NSPN_MRIProcessing) which is for MRI only. So can i start writing code by forking this repo.

**Comment by KirstieJane** [comment]
Hi @amitkunarj441 - you can fork it but I'm not receiving open contributions to that project.
